appveyor.yml

```diff
@@ -23,8 +23,7 @@ install:
   - cd %APPVEYOR_BUILD_FOLDER%
   - php -r "readfile('http://getcomposer.org/installer');" | php
   - php composer.phar install --no-interaction --no-progress
-  - appveyor DownloadFile https://phar.phpunit.de/phpunit.phar
 
 test_script:
   - cd %APPVEYOR_BUILD_FOLDER%
-  - php phpunit.phar --verbose
+  - php ./bin/phpunit --verbose
```

Comment:
I suggest moving the composer bins back to ``vendor/bin`` to be consistent with symfony-standard-edition and avoid conflicts with the project-specific bin which are also in the bin folder (making exclude rules much easier to write)